requirements.txt

```diff
@@ -1,3 +1,6 @@
 pytest
 flake8
 attrs>=18.1.0
+mypy>=0.600
```

Comment:
mypy should be added to travis as well at some point.

Although I assume right now it will throw many errors at us.